csm/core/blogic/const.py

```diff
@@ -970,3 +970,7 @@
 SWAGGER_JSON_URL = '/api/doc/swagger.json'
 SWAGGER_UI_STATICS_URL = '/api/static'
 
+# Cortx About Information
+INFORMATION_SERVICE = "information_service"
+ABOUT_INFO_RESOURCES = ['cluster', 'certificate']
+INFORMATION_AUTH_KEY = 'INFORMATION>auth'
```

Comment:
Not required for this API.

Reply:
Yes this is not required now.. I wil remove it